test/Microsoft.SqlTools.ServiceLayer.Test.Common/RequestContextMocking/RequestContextMocks.cs

```diff
@@ -73,5 +75,15 @@ public MockRequest()
         {
             Mock = RequestContextMocks.Create<T>(actual => result = actual);
         }
+
+        /// <summary>
+        /// Asserts that this request was successful
+        /// </summary>
+        /// <param name="handlerName">Name of the handler, recommended to use nameof(service.MyHandler), used in the failure message</param>
+        /// <param name="descriptor">Optional extra descriptor, parenthesized in the failure message</param>
+        public void AssertSuccess(string handlerName, string? descriptor = null)
+        {
+            Assert.IsTrue(this.Result.Success, $"{handlerName} {(descriptor != null ? $"(descriptor) " : String.Empty)}expected to succeed, but failed with error: '{this.Result.ErrorMessage}'");
```

Comment:
nit: space before `expected`

Reply:
Oops, good catch.  Stuck the optional space in the wrong spot.

Reply:
Wait, dangit.  This was right the first time: the spaces were after `{handlerName}` and (if descriptor was set) after `(descriptor)`.

But I did find that I did `(descriptor)` instead of `({descriptor})`, so not a total loss, lol.